webpack.config.js

```diff
@@ -30,6 +31,7 @@ module.exports = {
   plugins: [
     new MiniCssExtractPlugin(),
     process.env.REPLAY_BUILD_VISUALIZE && new Visualizer(),
+    new webpack.EnvironmentPlugin({ REPLAY_RELEASE: undefined }),
```

Comment:
Hmm, I'd expect this to mean that
```
release: process.env.REPLAY_RELEASE ? process.env.REPLAY_RELEASE : "development",
```
becomes
```
release: undefined ? undefined : "development",
```
and thus is always development, but I'm not totally sure on that.

Reply:
Oh dang I'm wrong on this one, I guess this uses the env var instead if it is present, missed that.